src/internal/clusterstate/v2.8.0_test.go

```diff
@@ -29,6 +58,23 @@ func Test_v2_8_0_ClusterState(t *testing.T) {
 	require.NoError(t, migrations.ApplyMigrations(ctx, db, migrationEnv, state_2_8_0))
 	require.NoError(t, migrations.BlockUntil(ctx, db, state_2_8_0))
 
+	// Get all collections commits.
+	commitsCol, err := v2_8_0.ListCommitsFromCollection(ctx, db)
+	require.NoError(t, err, "should be able to list commits from collection")
+	// Get all pfs.commits.
+	iter, err := pfsdb.ListCommit(ctx, db, nil, false)
+	require.NoError(t, err, "should be able to list commits from pfs.commits")
+	commitCmp := make(map[string]*pfs.CommitInfo)
+	require.NoError(t, stream.ForEach[pfsdb.CommitPair](ctx, iter, func(commitPair pfsdb.CommitPair) error {
+		commitCmp[pfsdb.CommitKey(commitPair.CommitInfo.Commit)] = commitPair.CommitInfo
+		return nil
+	}))
+	// compare collections commits to pfs.commits.
+	require.Equal(t, len(commitsCol), len(commitCmp), "all rows should have been migrated")
+	for _, info := range commitsCol {
+		commitsMatch(t, info, commitCmp[pfsdb.CommitKey(info.Commit)])
```

Comment:
Instead of custom function, try using the `cmp` package like the way it's done below. Either use `cmp.Diff` or `cmp.Equal`.